server/src/main/java/dev/slimevr/vr/trackers/VRTracker.java

```diff
@@ -23,6 +27,56 @@ public VRTracker(int id, String name, boolean hasRotation, boolean hasPosition)
 		super(id, name, name, hasRotation, hasPosition, null);
 	}
 
+	@Override
+	public boolean getRotation(Quaternion store) {
+		store.set(rotation);
+		// Don't adjust if tracker is HMD/Head (use raw rotation)
+		if (super.getBodyPosition() != TrackerPosition.HMD)
+			adjustInternal(store);
+		return true;
+	}
+
+	// TODO Reduce code duplication from IMUTracker.java
+	// Refactor all tracker classes into one?
+	protected void adjustInternal(Quaternion store) {
+		store.multLocal(attachmentFix);
+		store.multLocal(mountFix);
+		yawFix.mult(store, store);
+	}
+
+	@Override
+	public void resetFull(Quaternion reference) {
+		fixAttachment(rotation, reference);
+		fixYaw(rotation, reference);
+	}
+
+	@Override
+	public void resetYaw(Quaternion reference) {
+		fixYaw(rotation, reference);
+	}
+
+	private void fixAttachment(Quaternion sensorRotation, Quaternion reference) {
+		reference = reference.clone();
+		mountFix.fromAngles(0, reference.getYaw(), 0);
+
+		sensorRotation = sensorRotation.clone();
+		attachmentFix.set(sensorRotation.inverse());
```

Comment:
Do not cloe and then inverse. Either inverse() or clone and inverseLocal(). Also modifying argument variable is bad tone.